Developer-Guide.md

```diff
@@ -75,6 +76,22 @@ You need to install the following to build Kata Containers components:
 - `make`.
 - `gcc` (required for building the shim and runtime).
 
+# Individual component compatibility
+
+Using newer versions of individual components than the last offical released
```

Comment:
potential reword, and a typo s/offical/official/

`Using versions of individual components newer than the last official released`